packages/conduit-async/conduit-async.1.2.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer:   "anil@recoil.org"
+authors:      ["Anil Madhavapeddy" "Thomas Leonard" "Thomas Gazagnaire" "Rudi Grinberg"]
+homepage:     "https://github.com/mirage/ocaml-conduit"
+dev-repo:     "https://github.com/mirage/ocaml-conduit.git"
+bug-reports:  "https://github.com/mirage/ocaml-conduit/issues"
+tags:         "org:mirage"
+license:      "ISC"
+
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "jbuilder" {build & >="1.0+beta9"}
+  "core"
+  "ppx_sexp_conv"
+  "sexplib"
```

Comment:
Is this always needed or is it tied to ppx_sexp_conv and its need depends on the version of ppx_sexp_conv?

Reply:
it's used in the code (with `Sexplib.Sexp.to_string`)